manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShortestPathPlanningTest.scala

```diff
@@ -48,19 +48,29 @@ class ShortestPathPlanningTest extends DocumentingTest {
     synopsis("Shortest path finding in Cypher and how it is planned.")
     p("""Planning shortest paths in Cypher can lead to different query plans depending on the predicates that need
         |to be evaluated. Internally, Neo4j will use a fast bidirectional breadth-first search algorithm if the
-        |predicates can be evaluated whilst searching for the path. If the predicates need to inspect the whole path
-        |before deciding on whether it is valid or not, this fast algorithm cannot be used, and Neo4j will fall back to
-        |a slower exhaustive search for the path. The difference between these two can be in the order of magnitude,
-        |so it is important to ensure that the fast approach is used for time critical queries.""")
+        |predicates can be evaluated whilst searching for the path. Therefore, this fast algorithm will always
+        |be certain to return the right answer when there are universal predicates on the path; for example, when
+        |searching for the shortest path where all nodes have the `Person` label, or where there are no nodes with
+        |a `name` property.""".stripMargin)
+    p("""If the predicates need to inspect the whole path before deciding on whether it is valid or not, this fast
+        |algorithm cannot be relied on to find the shortest path, and Neo4j may have to resort to using a slower
+        |exhaustive depth-first search algorithm to find the path. This means that query plans for shortest path
+        |queries with non-universal predicates will include a fallback to running the exhaustive search to find
+        |the path should the fast algorithm not succeed. For example, depending on the data, an answer to a shortest
+        |path query with existential predicates -- such as the requirement that at least one node contains the property
+        |`name=Emil` -- may not be able to be found by the fast algorithm. In this case, Neo4j will fall back to using
```

Comment:
Suggest using `name='Charlie Sheen'`, or some other name from the example graph rather than "Emil". The name should probably be quoted.